bundle-ocata-s390x.yaml

```diff
@@ -190,7 +190,7 @@ services:
     num_units: 1
     options:
       worker-multiplier: 0.5
-      bridge-mappings: physnet1:br-ex
+      ext-port: eth1
```

Comment:
No reason other than what works in the mitaka bundle is ext-port. Will retry with physnet:eth1